src/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -224,6 +225,7 @@ See the LICENSE file in the project root for more information.
     <Message Text="Generating compatible native code. To optimize for size or speed, visit https://aka.ms/OptimizeCoreRT" Condition="$(_BuildingInCompatibleMode) == 'true'" Importance="high" />
 
     <Exec Command="&quot;$(IlcPath)\tools\ilc&quot; @&quot;$(NativeIntermediateOutputPath)%(ManagedBinary.Filename).ilc.rsp&quot;" />
+	<Exec Condition="$(VersionFile) != ''" Command="awk -f $(MSBuildThisFileDirectory)generateversionscript.awk &quot;$(ExportsFile)&quot; > &quot;$(VersionFile)&quot;"/>
```

Comment:
Instead of converting the files here and introducing dependency on awk, it would be better to generate the ExportsFile in the version script format here: https://github.com/dotnet/corert/blob/635cf21aca11265ded9d78d216424bd609c052f5/src/ILCompiler.Compiler/src/Compiler/ExportsFileWriter.cs#L35

Notice that we generate a different format for Windows vs. Unix already, so generating a different format for Windows vs. OSX vs. Unix should be fine.

Reply:
You are right, I returned the changes and separated the generate format the ExportsFile in order to preserve Windows and OSX formats.